ledger/ledger-api-integration-tests/src/test/itsuite/scala/com/digitalasset/platform/tests/integration/ledger/api/TransactionServiceIT.scala

```diff
@@ -324,7 +319,7 @@ class TransactionServiceIT
       "expose transactions to non-submitting stakeholders without the commandId" in allFixtures {
         c =>
           c.submitCreateAndReturnTransaction(
-              s"Checking_commandId_visibility_for_non-submitter_party-${runSuffix}",
+              s"Checking_commandId_visibility_for_non-submitter_party-${runCommandSuffix}",
```

Comment:
The leading `-` needs to be removed now from all command IDs.

Reply:
Done